### PR TITLE
Button and TextField added

### PR DESCRIPTION
**Pull request by florenciaferraiolo**
## Description

First commit, added button in main screen.

## Evidences

Captura de Pantalla 2022-03-31 a la(s) 13 24 52

## Tested on

Device: Nexus S

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -7,12 +7,25 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/textView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="Hola!"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: use strings res, do not hardcode

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -7,12 +7,25 @@
...
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/button"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent" />
+
+    <Button
+        android:id="@+id/button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Boton"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: use strings res, do not hardcode

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -7,12 +7,25 @@
...
+        app:layout_constraintTop_toBottomOf="@id/textView"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        tools:layout_editor_absoluteX="160dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: 1. why do you need these properties?
2. use dimens res, do not hardcode

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -7,12 +7,25 @@
...
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="Hola!"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/button"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent" />
+
+    <Button
+        android:id="@+id/button"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: put an id more representative please

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -7,12 +7,25 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/textView"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: put an id more representative please

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (florenciaferraiolo) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (florenciaferraiolo) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (florenciaferraiolo) [reply app/src/main/res/layout/activity_main.xml]: I forgot to remove them, they came as default

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (florenciaferraiolo) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 5 above)
Comment (florenciaferraiolo) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -7,12 +7,23 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/welcomeText"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: please follow the naming convention. The XML IDs shoud be in sneake_case. In this case should be welcome_text

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 6 above)
Comment (florenciaferraiolo) [reply app/src/main/res/layout/activity_main.xml]: done

**Review by SartiMau — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -7,12 +7,23 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/welcome_text_field"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="@string/welcome_text_field_text"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/check_button"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: Be careful with this, if you define an id with a + like here, you are creating a new reference. That should be used only when we are defining an id.

please remove that

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by mauroLonghi — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 7 above)
Comment (florenciaferraiolo) [reply app/src/main/res/layout/activity_main.xml]: good to know! done

**Review by SartiMau — APPROVED** [review]

**Review by mariano98montero — APPROVED** [review]

**Pull request merged**